src/codegen/resources/gateway/connection_properties

```diff
@@ -0,0 +1,3 @@
+$os	string	your operating system
```

Comment:
This is pretty inconsistent with how we did Routes and their Data classes. Should we consider moving all of their data classes here and only having the routes in .Java.

Reply:
I believe with the routes the inputs and such were not as consistent with how they are formatted in the discord docs.

If we can move them here as codegen, then we should.

Reply:
Then we should, maybe in another PR for Data once the rest of codegen works. Less priority